survey/static/survey/js/custom.js

```diff
@@ -0,0 +1,9 @@
+/* globals $ */
```

Comment:
Codacy has a javascript linting style here : https://app.codacy.com/manual/Pierre-Sassoulas/django-survey/pullRequest?prid=5139380

I don't know if it makes sense, but I don't have enough javascript to implement my own linting. Does it make sense to you ? Could you fix the warning if it's the case ?

Reply:
Yes, I see it now. Codacy expects another code style than I usually write, which is fine for me, I just needed to adapt the code accordingly.